# Regex pattern needs updating for newer versions of perl.
# This fix is from a newer version of this component.


--- XML-Simple-2.20/t/8_Namespaces.t_orig	2012-06-20 03:00:21.000000000 -0700
+++ XML-Simple-2.20/t/8_Namespaces.t	2016-04-05 09:07:27.004767331 -0700
@@ -122,16 +122,16 @@
 };
 
 $xml = XMLout($opt);
-like($xml, qr{
+like($xml, qr[
   ^\s*<opt
   (\s+{http://www.w3.org/2000/xmlns/}perl="http://www.perl.com/"
   |\s+{http://www.perl.com/}attr="value"
   |\s+bare="Beer!"){3}
   \s*>
-  \s*<{http://www.perl.com/}element\s*>data</{http://www.perl.com/}element\s*>
+  \s*<\{http://www.perl.com/\}element\s*>data</\{http://www.perl.com/\}element\s*>
   \s*</opt>
   \s*$
-}sx, 'clarkian names not converted to qnames on output by default');
+]sx, 'clarkian names not converted to qnames on output by default');
 
 
 # Confirm nsexpand option works on output
